plugins/Jobs.cpp

```diff
@@ -193,6 +194,57 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
         return true;
     }
 
+    // ----------------------------------------------------------------------
+    else if (SIEquals(request.methodLine, "CancelJob")) {
+        // - CancelJob(jobID)
+        //
+        //     Cancel a QUEUED, RUNQUEUED, FAILED job from a sibling.
+        //     (actually, it allows you to cancel non siblings, but you
```

Comment:
This is very confusing. I understand it because I have the background and I just re-read our discussion in the PR but I don't think this will be clear to anyone seeing this without context.

Since we're alerting in this case, I suggest removing this part of the comment (the part in parenthesis)